.github/workflows/deploy.yml

```
@@ -114,6 +114,8 @@ jobs:
           kubectl set image -n ${KUBE_NAMESPACE} \
           deployment/peoplefinder \
           webapp="${{ vars.ECR_URL }}:$SHA"
```

I think you need to add back slashes like the previous lines have
```suggestion
          webapp="${{ vars.ECR_URL }}:$SHA" \
```